From 013aa2949d51fe04091d7841ead633e95feba3eb Mon Sep 17 00:00:00 2001
From: xieyi <xieyi@rock-chips.com>
Date: Thu, 27 Sep 2012 10:03:24 +0800
Subject: [PATCH] rk2928: synchronize rk2926_m713_defconfig with a720

commit e47737f792d3b3317f267f6647207e311d0fce4e
Author: kfx <kfx@rock-chips.com>
Date:   Wed Sep 26 21:00:07 2012 +0800

rk2928: cpu 1G && rk2928_a720_defconfig(usb host && gpio device && i2c device support)
---
 arch/arm/configs/rk2926_m713_defconfig | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/configs/rk2926_m713_defconfig b/arch/arm/configs/rk2926_m713_defconfig
index 7129a42e739a..3279495883c7 100644
--- a/arch/arm/configs/rk2926_m713_defconfig
+++ b/arch/arm/configs/rk2926_m713_defconfig
@@ -248,10 +248,12 @@ CONFIG_GSENSOR_DEVICE=y
 # CONFIG_HW_RANDOM is not set
 CONFIG_I2C=y
 # CONFIG_I2C_COMPAT is not set
+CONFIG_I2C_CHARDEV=y
 CONFIG_I2C0_CONTROLLER_RK30=y
 CONFIG_I2C1_CONTROLLER_RK30=y
 CONFIG_I2C2_CONTROLLER_RK30=y
 CONFIG_ADC_RK30=y
+CONFIG_GPIO_SYSFS=y
 CONFIG_EXPANDED_GPIO_NUM=0
 CONFIG_EXPANDED_GPIO_IRQ_NUM=0
 CONFIG_SPI_FPGA_GPIO_NUM=0
@@ -367,6 +369,7 @@ CONFIG_USB_SERIAL_OPTION=y
 CONFIG_USB_GADGET=y
 CONFIG_USB20_HOST=y
 CONFIG_USB20_OTG=y
+CONFIG_DWC_OTG_BOTH_HOST_SLAVE=y
 CONFIG_MMC=y
 CONFIG_MMC_UNSAFE_RESUME=y
 CONFIG_MMC_EMBEDDED_SDIO=y
-- 
2.35.3

